addons/binding/org.openhab.binding.homematic/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="homematic"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Homematic Bridge -->
+	<bridge-type id="bridge">
+		<label>Homematic Bridge</label>
+		<description>The Homematic bridge represents a Homematic gateway</description>
+
+		<config-description>
+			<parameter name="gatewayAddress" type="text" required="true">
+				<context>network_address</context>
+				<label>Gateway Address</label>
+				<description>Network address of the Homematic gateway</description>
+			</parameter>
+            <parameter name="gatewayType" type="text">
+                 <label>Gateway Type</label>
+                 <description>Hint for the binding to identify the gateway type (default = auto)</description>
+                 <limitToOptions>true</limitToOptions>
+                 <default>auto</default>
+                 <options>
+                     <option value="auto">Auto</option>
+                     <option value="ccu">CCU</option>
+                 </options>
+            </parameter>
+			<parameter name="callbackHost" type="text">
```

Comment:
here is a mix of spaces and tabs in the file - seems that our auto-formatter only takes care of java files, but not xml